Makefile

```diff
@@ -97,6 +100,9 @@ size:
 build-esp32-bin:
 	cargo +esp espflash save-image --package micro-rdk-server --merge --chip esp32 target/xtensa-esp32-espidf/micro-rdk-server-esp32.bin -T micro-rdk-server/esp32/partitions.csv -s 4mb  --bin micro-rdk-server-esp32 --target=xtensa-esp32-espidf  -Zbuild-std=std,panic_abort --release
 
+build-esp32-bin-provisioning:
```

Comment:
not sure how to make this less copy-paste

Reply:
like `make build-esp32-bin --features=provisioning`? I think we can using Make flags, but I'm not as familiar with it compared to a command runner like [`just`](https://github.com/casey/just).

Or did you mean just leave this off for now entirely?